compiler/rustc_parse_format/src/lib.rs

```diff
@@ -813,101 +813,101 @@ fn find_skips_from_snippet(
     snippet: Option<string::String>,
     str_style: Option<usize>,
 ) -> (Vec<usize>, bool) {
+    if str_style.is_some() {
+        return (vec![], true);
+    }
```

Comment:
Should this be below the next block to keep the same behaviour for the returned `bool`?